### PR TITLE
docs(readme): add keyboard shortcuts - #244

### PR DESCRIPTION
**Pull request by 98lenvi**
# Issue #244 
Updated readme to show the new keyboard shortcuts
### Changes
- The readme has the following text added just before the ### Demo section
------------------------------------------------
### Keyboard shortcuts

The keyboard shortcuts are as follows

Windows/Linux:

- Common actions
  - `CTRL` + `C` - Copy
  - `CTRL` + `X` - Cut
  - `CTRL` + `V` - Paste
  - `CTRL` + `Z` - Undo
  - `CTRL` + `SHIFT` + `Z` - Redo
  - `CTRL` + `Y` - Repeat last action

- Text formatting
  - `CTRL` + `B` - Bold
  - `CTRL` + `I` - Italicize 
  - `CTRL` + `Q` - Block quotes
  - `CTRL` + `SHIFT` + `9` - Code

- Paragraph formatting
  - `CTRL` + `SHIFT` + `7` - Numbered list
  - `CTRL` + `SHIFT` + `8` - Bulleted list

Mac:

- Common actions
  - `⌘` + `C` - Copy
  - `⌘` + `X` - Cut
  - `⌘` + `V` - Paste
  - `⌘` + `Z` - Undo
  - `⌘` + `SHIFT` + `Z` - Redo
  - `⌘` + `Y` - Repeat last action

- Text formatting
  - `⌘` + `B` - Bold
  - `⌘` + `I` - Italicize 
  - `⌘` + `Q` - Block quotes
  - `⌘` + `SHIFT` + `9` - Code

- Paragraph formatting
  - `⌘` + `SHIFT` + `7` - Numbered list
  - `⌘` + `SHIFT` + `8` - Bulleted list
----------------------------------------------------------

### Flags
- None

### Related Issues
- Issue #244 
- Pull Request #267

### CODE REVIEW

**Comment by 98lenvi** [comment]
In PR #285, the shortcut for blockquotes was changed to `CTRL` + `SHIFT` + `.` , I'll have to update it here

In PR #294, the shortcut for code is changed to `CTRL` + `SHIFT` + `9`

**Comment by elit-altum** [comment]
> 
> 
> These two do not work for me. Is that to be addressed in another PR?
> ⌘ + Q - Block quotes
> ⌘ + SHIFT + 9 - Code

Just checked for the code shortcut. It is working for me.

**Comment by irmerk** [comment]
> These two do not work for me. Is that to be addressed in another PR?
> ⌘ + Q - Block quotes
> ⌘ + SHIFT + 9 - Code

Right, block quotes is [here](https://github.com/accordproject/markdown-editor/blob/development/src/SlateAsInputEditor/index.js#L397): `MOD + SHIFT + .`

[Code shortcut](https://github.com/accordproject/markdown-editor/blob/development/src/SlateAsInputEditor/index.js#L395) should be working, though...

**Comment by irmerk** [comment]
> Just checked for the code shortcut. It is working for me.

It isn't working for me in the demo, but that's a different issue that can be addressed in a different PR.

**Comment by 98lenvi** [comment]
Well, I updated the shortcut for the blockquote, by doing so I just pushed a commit and due to a [known GitHub bug](https://github.com/isaacs/github/issues/1157) it looks like I dismissed the reviews(Which I didn't).

**Comment by irmerk** [comment]
@98lenvi no we have the settings set to dismiss reviews when new commits are pushed.

File: README.md [hunk 1]
```diff
@@ -46,6 +46,50 @@ The editor includes a formatting toolbar.
...
+
+Mac:
+
+- Common actions
+  - `⌘` + `C` - Copy
+  - `⌘` + `X` - Cut
+  - `⌘` + `V` - Paste
+  - `⌘` + `Z` - Undo
+  - `⌘` + `SHIFT` + `Z` - Redo
+  - `⌘` + `Y` - Repeat last action
```
Comment (irmerk) [inline README.md]: I don't think this is a current shortcut we are supporting, we can remove it.

Comment (Michael-Grover) [reply README.md]: this code snippet shows four shortcuts on my screen, are you referring to just "Repeat last action"?

Comment (irmerk) [reply README.md]: Yes sorry I highlighted line 81 but it shows above that line.

Comment (98lenvi) [reply README.md]: I've removed it, Please check it now.

**Comment by irmerk** [comment]
@98lenvi would you be able to recreate this for `master`? I'm scrapping the `development` branch.

**Pull request merged**